src/Extension/FluentExtension.php

```diff
@@ -611,9 +612,13 @@ protected function localiseManipulationTable(&$manipulation, $table, $localeTabl
      *
      * @param SQLSelect $query
      * @param DataQuery $dataQuery
+     * @deprecated 4.8.0 Will be removed without equivalent functionality
      */
     public function augmentDataQueryCreation(SQLSelect $query, DataQuery $dataQuery)
     {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice('4.8.0', 'Will be removed without equivalent functionality');
+        });
```

Comment:
This needs to be wrapped with `Deprecation::withNoReplacement()` because this method will be called by `DataQuery` and we don't want to (by default) output notices about things people can't control in their own projects.

Reply:
are we keeping data query params or are they going away?

Reply:
All of those parameters are still valid - but this method is deprecated _for this module specifically_ for this extension because the method isn't in fluent 5. The actual hook is still valid in CMS 5, it's just not in use _for this module specifically_ from fluent 5 onwards.

Reply:
Oh, it was removed at some point. I don't think I noticed it lol.

Reply:
It's in fluentobjecttrait :) it's the mystery moving code,

Reply:
Oohhhhh nevermind then I'll undeprecate that one.